docs/source/developer_guide/core/internals.rst

```diff
@@ -61,11 +61,11 @@ The :py:class:`~aiida.orm.nodes.Node` can be annotated with labels, description
 
 *Label management:*
 
-- :py:meth:`~aiida.orm.nodes.Node.label` returns the label of the node and can be used as a setter property.
+- :py:attr:`~aiida.orm.nodes.Node.label` returns the label of the node. The setter can be used to *change* the label, e.g. ``mynode.label = "new label"``.
```

Comment:
Separate line for each complete single sentence.

Reply:
Maybe even more explicitly, leave out "setter" to instead write something like:

```suggestion
- :py:attr:`~aiida.orm.nodes.Node.label` returns the label of the node.
  It can also be used to *change* the label, e.g. ``mynode.label = "new label"``.
```